sbomgrader/__main__.py

```diff
@@ -133,5 +111,52 @@ def main():
     exit(1)
 
 
+def convert():
+    parser = ArgumentParser("sbomconv")
+    parser.add_argument(
+        "input",
+        type=Path,
+        help="SBOM File to convert. Currently supports JSON.",
+        nargs="?",
+    )
+    parser.add_argument(
+        "--list-maps",
+        "-l",
+        action="store_true",
+        default=False,
+        help="List available default translation maps and exit.",
+    )
+    parser.add_argument(
+        "--output-format",
+        "-f",
+        choices=[v.value for v in SBOMFormat],
+        required=not ("--list-maps" in sys.argv or "-l" in sys.argv),
```

Comment:
Not a blocker but I think this would be a reasonable replacement https://docs.python.org/3/library/argparse.html#mutual-exclusion